File: Beecrowd/Python/bee1132.py
first = int(input())
second = int(input())
total = 0
count = first

if first > second:
    aux = second
    second = first
    first = aux

for i in range(first, second + 1):
    if not(i % 13 == 0):
        total += i

print(total)

File: Beecrowd/C/bee1132.c
#include <stdio.h>

int main() {
  int aux, x, y, total = 0;

  scanf("%d\n%d", &x, &y);

  if (y < x) {
    aux = y;
    y = x;
    x = aux;
  }

  for (int i = x; i <= y; i++) {
    if (i % 13 != 0)
      total += i;
  }

  printf("%d\n", total);

  return 0;
}
